lib/liquid_voting_web/resolvers/voting.ex

```diff
@@ -45,17 +44,24 @@ defmodule LiquidVotingWeb.Resolvers.Voting do
   def create_vote(_, %{participant_email: email, proposal_url: _, yes: _} = args, %{
         context: %{organization_id: organization_id}
       }) do
-    case Voting.upsert_participant(%{email: email, organization_id: organization_id}) do
-      {:error, changeset} ->
-        {:error,
-         message: "Could not create vote with given email",
-         details: ChangesetErrors.error_details(changeset)}
+    Tracer.with_span "#{__MODULE__} #{inspect(__ENV__.function)}" do
+      Tracer.set_attributes([
+        {:request_id, Logger.metadata()[:request_id]},
+        {:vars, binding()}
```

Comment:
same question here - which vars are being sent? we only want  a few key ones, right

Reply:
See reply above.